### PR TITLE
[sonic-mgmt]: Upgrade scapy

### PR DESCRIPTION
**Pull request by Pterosaur**
Signed-off-by: Ze Gan <ganze718@gmail.com>

#### Why I did it
The version of scapy in apt source is out-of-date to MACsec.

#### How I did it
Upgrade scapy by pip

#### How to verify it
Check the version of scapy that should be 2.4.5
```
root@5e327a6ebcb5:/# python
Python 2.7.17 (default, Feb 27 2021, 15:10:58)
[GCC 7.5.0] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import scapy
>>> scapy.__version__
'2.4.5'

```

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106

### CODE REVIEW

**Comment by lguohan** [comment]
have you do some more ptf testing? will this cause regression for the nightly?

File: dockers/docker-sonic-mgmt/Dockerfile.j2 [hunk 1]
```diff
@@ -88,7 +88,8 @@ RUN pip install cffi==1.10.0 \
     && rm -fr nanomsg-1.0.0 \
     && rm -f 1.0.0.tar.gz  \
     && pip install nnpy    \
-    && pip install dpkt
+    && pip install dpkt    \
+    && pip install scapy --upgrade
```
Comment (wangxin) [inline dockers/docker-sonic-mgmt/Dockerfile.j2:92]: What is the relationship between this pip installed `scapy` and the apt installed `python-scapy` above?

**Review by wangxin — DISMISSED** [review]

File: dockers/docker-sonic-mgmt/Dockerfile.j2 (hunk 1 above)
Comment (Pterosaur) [reply dockers/docker-sonic-mgmt/Dockerfile.j2]: python-ptf will check whether python-scapy was installed, but the python-scapy is the previous version that cannot meet our requirement. So I install python-ptf firstly and then upgrade scapy by pip.

**Review by wangxin — APPROVED** [review]

**Pull request merged**